apps/web/app/dashboard/_components/DashboardProvider.tsx

```diff
@@ -20,15 +24,23 @@ export default function DashboardProvider(): JSX.Element {
     enabled: Boolean(user?._id),
   });
 
+  useEffect(() => {
+    if (!user?._id) {
+      router.push("/login");
```

Comment:
```suggestion
      router.push(PAGE_NAME.SIGN_IN);
```
상수로 관리중인 페이지 경로를 넣어주는게 더 좋을거같아~